### ISSUE TITLE
Document requirement for JRE

### ISSUE THREAD

**Issue by jonrkarr** [GINsim/GINsim-python#14]

**Comment by aurelien-naldi** [comment]
Do you mean on the Python module? We should indeed clarify it there, I guess it is missing as we mostly use the conda package which explicitly  depends on a JRE

**Comment by jonrkarr** [comment]
Yes, JRE is an indirect requirement to use this package. For example, a minimal python docker image plus pip installation of this package wouldn't be sufficient to use this package because a JRE wouldn't be available. A JRE would need to be installed separately.
